Update azure-sentinel-billing.md

nayef-yassin opened the pull request:
Adding a new subsection for the new AMA based Windows Security Events connector which can help optimize costs. I am not sure my formatting is correctly done in the md file and would appreciate any feedback/guidance. 

I was not sure how to link the new header (Use Data Collection Rules for your Windows Security Events) to the new section I added.

nayef-yassin committed:
Update azure-sentinel-billing.md

Adding a new subsection for the new AMA based Windows Security Events connector which can help optimize costs. I am not sure my formatting is correctly done in the md file and would appreciate any feedback/guidance. 

I was not sure how to link the new header (Use Data Collection Rules for your Windows Security Events) to the new section I added.

batamig commented on articles/sentinel/azure-sentinel-billing.md:
```diff
@@ -198,6 +199,12 @@ With ADX, you can store data at a lower price, but still explore the data using
 
 For more information, see [Integrate Azure Data Explorer for long-term log retention](store-logs-in-azure-data-explorer.md).
 
+#### Use Data Collection Rules for your Windows Security Events
```
@nayef-yassin use title caps here:
#### Use data collection rules for your Windows security events

batamig commented on articles/sentinel/azure-sentinel-billing.md:
```diff
@@ -116,6 +116,7 @@ Manage data ingestion and retention:
 - [Optimize Log Analytics costs with dedicated clusters](#optimize-log-analytics-costs-with-dedicated-clusters).
 - [Separate non-security data in a different workspace](#separate-non-security-data-in-a-different-workspace).
 - [Reduce long-term data retention costs with Azure Data Explorer (ADX)](#reduce-long-term-data-retention-costs-with-adx).
+- Use Data Collection Rules for your Windows Security Events.
```
@nayef-yassin add an xref like this:

```
- [Use data collection rules for your Windows security events](#use-data-collection-rules-for-your-windows-security-events)
```

nayef-yassin replied:
I see your pasteable xref method now - thank you so much Batami!

batamig commented on articles/sentinel/azure-sentinel-billing.md:
```diff
@@ -198,6 +199,12 @@ With ADX, you can store data at a lower price, but still explore the data using
 
+#### Use Data Collection Rules for your Windows Security Events
+
+The [Windows Security Events connector](https://docs.microsoft.com/en-us/azure/sentinel/connect-windows-security-events?tabs=LAA) lets you stream security events from any Windows server (physical or virtual, on-premises or in any cloud) connected to your Azure Sentinel workspace. The new version of this connector is currently in preview and is based on the new Azure Monitor Agent (AMA). The Azure ...
```
@nayef-yassin please update with these edits:

```
The [Windows Security Events connector](/azure/sentinel/connect-windows-security-events?tabs=LAA) enables you to stream security events from any Windows server that's connected to your Azure Sentinel workspace, including physical, virtual, or on-premises servers, or in any cloud. This connector includes support for the Azure Monitor Agent (AMA), which uses data collection rules (DCR) to define the data to collect from each agent. 

Data collection rules enable you to manage collection settings at scale, while still allowing unique, scoped configurations for subsets of machines. For more information, see [Configure data collection for the Azure Monitor agent](/azure/azure-monitor/agents/data-collection-rule-azure-monitor-agent).

Besides for the predefined sets of events that you can select to ingest, such as *All events*, *Minimal*, or *Common*, data collection rules enable you to build custom filters and select specific events to ingest. The Azure Monitor Agent uses these rules to filter the data at the source, and then ingest only the events you've selected, while leaving everything else behind. Selecting specific events to ingest can help you optimize your costs and save more.
```

batamig reviewed (changes requested):
hi @nayef-yassin i put some editing comments in, and also some code for adding the xref at the top.
please implement these in the source and we'll push this through - thank you!
(you should be able to browse directly back to this file from the PR. please reach out to me or yechiel (yechiel is more likely to be online when you are) if you need help.

nayef-yassin committed:
Update azure-sentinel-billing.md

Updated with changes per Batami's review. My cross reference might not be working - would appreciate reviewing that.

nayef-yassin committed:
Create azure-sentinel-billing.md

Updated with changes per Batami's review. My cross reference might not be working - would appreciate reviewing that.

ktoliver committed:
PR review edit
